src/Styleguide/Components/__stories__/Tabs.story.tsx

```diff
@@ -58,12 +58,38 @@ storiesOf("Styleguide/Components", module).add("Tabs (Simple)", () => {
         </Tabs>
       </Section>
       <Section title="With separator">
-        <Tabs justifyContent="center" separator={<Box pr={"20px"}>*</Box>}>
+        <Tabs justifyContent="center" separator={<Box mx={1}>♞</Box>}>
           <Tab name="About" />
           <Tab name="Pricing" />
           <Tab name="Condition" />
         </Tabs>
       </Section>
+      <Section title="With superscript">
+        <Tabs justifyContent="center">
+          <Tab
+            name={
+              <React.Fragment>
+                Open<Sup>100</Sup>
+              </React.Fragment>
+            }
+          />
+          <Tab
+            name={
+              <React.Fragment>
```

Comment:
Instead of writing `<React.Fragment>` you can use `<></>`. It's a shorthand for fragment. https://reactjs.org/docs/fragments.html#short-syntax